src/KeyVault/KeyVault/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 * Removed two aliases: `New-AzKeyVaultCertificateAdministratorDetails` and `New-AzKeyVaultCertificateOrganizationDetails`
 * When creating a key vault, soft delete is now enabled by default
 * When creating a key vault, network rules can be set to govern the accessibility from specific network locations
+* `Get-AzKeyVaultKey` supports downloading a public key in PEM format
```

Comment:
Here miss the changelog of `New-AzKeyVaultKey`.

Reply:
Thanks I've made the change